tests/cpp-tests/Classes/AppDelegate.cpp

```diff
@@ -141,7 +143,9 @@ void AppDelegate::applicationWillEnterForeground()
     
     Director::getInstance()->startAnimation();
     // resume audioEngine, otherwise the opensl audioPlayer will always be suspended.
+ #if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
```

Comment:
Why change this?
This will make other platforms not trigger the onEnterbackground & onEnterForeground callbacks.

Reply:
There is not these code in 3.17.2 tag, so if OHOS needs it, it's fine here.